hazelcast/src/main/resources/release_notes.txt

```diff
@@ -1,96 +1,121 @@
+
+## Breaking Changes
+
+* Introduced a check to control the versions of Hazelcast Platform members and Hazelcast CLI are matched. Previously, it was possible to submit a job using a different version of CLI from the member version. Generally, it would work when there were no change in the serialized form of pipeline. Now, Hazelcast fails earlier in this case. [#22224]
+* Removed the `BETA` annotations from the compact serialization and `GenericRecord` related classes, i.e., they are stable. Now, compact serialization is enabled by default; the `enabled` attribute within the `compact-serialization` configuration block does not exist anymore. [#21997]
+
+## Enhancements
+
+### SQL Engine
+
+* SQL storage now is able to replicate data to the newly joined members in the cluster. [#21632]
+* Added support of the `Calc` operator to merge `Project` and `Filter`. [#17058]
+
+### Distribution
+
+* Removed `hazelcast-hibernate53` dependency from the main Hazelcast Platform distribution as it is not needed anymore. [#22282]
+* Minor versions of Hazelcast Platform are now released as X.Y.0, instead of the previous X.Y versioning scheme. [#22218]
+
+### Serialization
+
+* Added support of `List`, `ArrayList`, `Set`, `HashSet`, `Map`, and `HashMap` for the zero-config serializers. [#21980]
+* Added a check to the array of `Compact` and `GenericRecord` object fields, that does not allow different item types and schemas in such fields. [#21958]
+* Moved the `GenericRecord` and `GenericRecordBuilder` interfaces to the new `serialization.genericrecord` package. [#21955]
+* In case there is a field type that is not supported by the reflective serializer, now Hazelcast fails with an exception; all JDK classes are now excluded from the zero-config serialization, meaning, they cannot be used as types, field types, or array component types in the reflective serializers. [#21918]
+* Hazelcast now does not provide methods to read a default value in case of a missing field in the data. Instead, the following method has been introduced in `CompactReader` to check the existence of a field with its name and kind.+
+
+    FieldKind getFieldKind(String fieldName);
+
+    You can use this method for fields that have changed or have a potential to change in the future. [#21876]
+* Moved the `type-name` and `class` configuration elements into the `compact-serialization` block. Removed the `registered-classes` element. [#21861]
+* Renamed the `cloneWithBuilder()` method as `newBuilderWithClone()` in the `GenericRecord` class. [#21730]
+* Added support for `char` fields in the compact serialization format. With this, `char`, `char[]`, `Character`, and `Character[]` fields are now supported in the reflective compact serializers. [#21054]
+* Hazelcast now does not allow to create SQL mappings containing the compact format, when the compact serialization is not enabled, and provides a meaningful error; previously this situation was producing confusing errors. [#20998]
```

Comment:
I think this is no longer true, since the compact is enabled by default now. I believe we should remove this item altogether